docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -263,7 +263,7 @@ containerized workloads. At the completion of `kubestellar init` the current wor
 "root".
 
 6. Creates the space provider objects to be used by the space manager. Need to pass
-in two environment variables - SPACE_MANAGER_KUBECONFIG and IN_CLUSTER. SPACE_MANAGER_KUBECONFIG
+in two environment variables - SM_CONFIG and IN_CLUSTER. SM_CONFIG
```

Comment:
Why does `kubestellar init` have an `IN_CLUSTER` parameter? I think that we can safely assume that it is invoked nearby.